pkgs/tools/misc/bepasty/default.nix

```diff
@@ -1,35 +1,37 @@
-{ python3
-, lib
-}:
+{ python3, lib }:
 
 let
   python = python3.override {
     self = python;
-    packageOverrides = self: super : {
-      xstatic-bootstrap = super.xstatic-bootstrap.overridePythonAttrs(oldAttrs: rec {
-        version = "3.3.7.1";
-        src = oldAttrs.src.override {
-          inherit version;
-          sha256 = "0cgihyjb9rg6r2ddpzbjm31y0901vyc8m9h3v0zrhxydx1w9x50c";
-        };
-      });
+    packageOverrides = self: super: {
+      xstatic-bootstrap = super.xstatic-bootstrap.overridePythonAttrs
+        (oldAttrs: rec {
+          version = "3.3.7.1";
+          src = oldAttrs.src.override {
+            inherit version;
+            sha256 = "0cgihyjb9rg6r2ddpzbjm31y0901vyc8m9h3v0zrhxydx1w9x50c";
+          };
+        });
```

Comment:
The override must be deleted. bepasty uses now boostrap4 (which is what we have in the repo)